types/TestResult.js

```diff
@@ -105,14 +139,36 @@ export type TestResult = {|
   testResults: Array<AssertionResult>,
 |};
 
+export type FormattedTestResult = {|
```

Comment:
It's nice we are adding these. Can we reconcile the types in `jest-editor-support/src/types.js` with the ones here in a follow-up PR? It seems like you are hoisting up these types to here and we can probably get rid of the ones in jest-editor-support.